### PR TITLE
WIP: Adjust Webpage for release

### LINKED ISSUE #11: Bundled documentation outdated
The documentation bundled with CARAT is outdated in some parts. In particular, the table with the numbers of crystallographic groups of small dimensions contains wrong entries for dimension 6. These have been corrected only on the CARAT home page.

### PR DESCRIPTION
**Pull request by DominikBernhardt**
This adjusts information on the Website for Carat.

This is still work in progress, I'm happy to get feedback. 
Resolves #11 .

### CODE REVIEW

**Comment by DominikBernhardt** [comment]
@fingolfin I listed you as contributor. I am unsure what to do there.
Also, I listed Prof. Plesken as author as he inspired and developed a lot of the underlying mathematics, but the code was written by the other two people. Any thoughts?

**Comment by DominikBernhardt** [comment]
@fingolfin : I did a lot of work on the website and would appreciate it if you had a look.
There is still a few things I need to do but I would prefer for this PR to be merged first so there is a "base" as this website is live already. In particular, I also worked on documentation subpage and made a bunch of changes. 
Things that still need to be done:

- Interactive table of low dimensional Bierbach groups does not work
- Provide download links
- What are the dependencies? Which gcc do we need?

**Comment by gaehler** [comment]
Please note issue #11: A more recent version of the web page (found on web.archive.org) has corrected numbers of crystallographic groups in dimension 6.

**Comment by fingolfin** [comment]
@DominikBernhardt frankly, I think you should just merge this (whether you squash or not is up to you, either is fine by me). Then it'll be visible "live", and I'll be happy to look at the resulting website and give feedback on that.

File: doc/examples/.xvpics/Ex13.gif [hunk 1]
```diff
@@ -0,0 +1,5 @@
+P7 332
+#IMGINFO:828x1170 RGB (6847 bytes)
```
[the remark this answers is not in the document]
Comment (DominikBernhardt) [reply doc/examples/.xvpics/Ex13.gif]: No, good catch, I'll remove it.

**Comment by DominikBernhardt** [comment]
@fingolfin Squashed and merged.

**Pull request merged**